frameworks/person-escort-record/questions/medication-while-moving.yml

```diff
@@ -0,0 +1,13 @@
+type: radio
```

Comment:
this question does not belong to a section or is a dependent on another quetsion

Reply:
Great spot :) It's good the importer spots that too 💯